src/app/home/home.component.ts

```diff
@@ -142,9 +142,9 @@ export class HomeComponent implements OnInit, AfterViewInit, OnDestroy {
           }
           return notifications;
         }).filter(nt  => {
-          return nt['user'] === user_id;
+          return nt.user === user_id;
         });
-        this.notifications['count_unread'] =  cnt;
+        this.notifications.count_unread = cnt;

```

Comment:
if i'm correct. Notifications are called by couchdb alldocs which will return array not object. So this.notifications will be assigned by normal array of documents. And this code is still trying to mix array and object under notifications.

Reply:
Yes, this is correct.  Thinking about this more, we do not even use this count in the component.  We should remove it until we do.